tests/__snapshots__/SettingsMigrationTest__it_migrates_pre_v5.0.0_settings_with_data_set_filled__1.json

```diff
@@ -1,476 +1,496 @@
 {
-  "account": {
-    "id": "account",
-    "apiKey": "b183f9372d6f9bd050418e31cc40965360ca814d",
-    "apiKeyValid": true
-  },
-  "order": {
-    "id": "order",
-    "barcodeInNote": true,
-    "barcodeInNoteTitle": "T&T:",
-    "conceptShipments": true,
-    "emptyDigitalStampWeight": 200,
-    "emptyMailboxWeight": 0,
-    "emptyParcelWeight": 1100,
-    "emptyPackageSmallWeight": 0,
-    "orderMode": false,
-    "processDirectly": "-1",
-    "saveCustomerAddress": true,
-    "sendNotificationAfter": -1,
-    "sendReturnEmail": false,
-    "shareCustomerInformation": true,
-    "statusOnLabelCreate": "wc-processed",
-    "statusWhenDelivered": -1,
-    "statusWhenLabelScanned": -1,
-    "trackTraceInAccount": true,
-    "trackTraceInEmail": true
-  },
-  "label": {
-    "id": "label",
-    "description": "[ORDER_ID]- [PRODUCT_NAME] - [PRODUCT_ID]- test",
-    "format": "a6",
-    "output": "open",
-    "position": [1, 2, 3, 4],
-    "prompt": true
-  },
-  "customs": {
-    "id": "customs",
-    "countryOfOrigin": "NL",
-    "customsCode": "1234",
-    "packageContents": "2"
-  },
-  "checkout": {
-    "id": "checkout",
-    "allowedShippingMethods": {
-      "-1": [],
-      "package": [
-        "flat_rate:32",
-        "free_shipping",
-        "local_pickup",
-        "table_rate:5:2"
-      ],
-      "package_small": [],
-      "mailbox": ["table_rate:5:1"],
-      "digital_stamp": ["flat_rate:30", "flat_rate:31"],
-      "letter": ["flat_rate:33"]
+    "account": {
```

Comment:
prettier is niet gerund

Reply:
Check, prettier zou nu toegepast moeten zijn.